internal/services/servicebus/servicebus_queue_data_source.go

```diff
@@ -37,25 +38,6 @@ func dataSourceServiceBusQueue() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
```

Comment:
This needs to be `Required` but updated to `Optional` in the `FivePointOh` block below

Reply:
Roger, updated